src/target/linux.rs

```diff
@@ -1,12 +1,39 @@
+use crate::target::thread::Thread;
+use crate::target::unix::{self, UnixTarget};
 use nix::unistd::{getpid, Pid};
+use procfs::process::Process;
 use std::{
     fs::File,
     io::{BufRead, BufReader},
     marker::PhantomData,
     mem,
 };
 
-use crate::target::unix::{self, UnixTarget};
+struct LinuxThread {
+    name: String,
+    id: usize,
+}
+
+impl LinuxThread {
+    fn new(name: &str, id: usize) -> LinuxThread {
+        LinuxThread {
+            name: name.to_string(),
+            id,
+        }
+    }
+}
+
+impl Thread for LinuxThread {
+    type ThreadId = usize;
```

Comment:
usize doesn't make sense for this. Please use whatevee type `task.tid` is.

Reply:
fixed